src/Core/NuGet/Core/Package.nuspec

```diff
@@ -67,5 +72,10 @@
     <file src="$configuration$\Core\Managed\Net46\Microsoft.ApplicationInsights.pdb" target="lib\net46" />
     <file src="$configuration$\Core\Managed\Net46\Microsoft.ApplicationInsights.xml" target="lib\net46" />
 
+    <!-- wp8: Binaries for Windows Phone 8.0 and 8.1 (Silverlight Runtime) projects -->
+    <file src="$configuration$\Core\Managed\DummyCore\Wp8Files\Microsoft.ApplicationInsights.dll" target="lib\wp8" />
+
+    <!-- portable-win81+wpa81: Windows Runtime projects (Phone & Store, Managed)-->
+    <file src="$configuration$\Core\Managed\DummyCore\Portable81Files\Microsoft.ApplicationInsights.dll" target="lib\portable-win81+wpa81" />
```

Comment:
Try this one as well:
<file src="$configuration$\Core\Managed\DummyCore\Portable81Files\Microsoft.ApplicationInsights.dll" target="lib\uap10.0" />